app/views/vacancies/_search.html.slim

```diff
@@ -13,12 +13,21 @@ ruby:
     {
       title: t("jobs.filters.nqt_suitable"),
       key: "nqt_suitable",
-      hidden_text: t("jobs.filters.nqt_suitable_hidden_prefix"),
```

Comment:
is @jesse ok with this? i seem to remember i added it after discussion with himn so the labels and text in the remove buttons wasnt so verbose and hence visually a bit ugly

Reply:
@alexbowen Excellent spot! The new label text includes the phrasing previously in the hidden prefix, so it's always part of the label:
```

Reply:
if sticking with the visually hidden prefix, can be bit more generic than `nqt_suitable_hidden_prefix` for sure, but the ui started looking a bit odd with the super long labels but needed to be descriptive for screen readres

Reply:
I do think it's a bit long, but the new design definitely has the whole "Only show results" shebang visible to all users, with nothing only for screenreaders